### PR TITLE
create homepage for IBooking

### PR DESCRIPTION
**Pull request by chrisudf**
image
image

The homepage of IBooking includes all the service categories, banner image, layout, and simple guide.

### CODE REVIEW

File: src/components/Homepage/HowItWork.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        </Col>
+        <Col md={3}>
+          <FontAwesomeIcon icon={faCheckCircle} className={Styles.guideIcon} />
+          <h5 className={Styles.guideContentTitle}>Confirm Booking.</h5>
+          <p>We will confirm your booking along with your instructions via secure transaction.</p>
+        </Col>
+        <Col md={3}>
+          <FontAwesomeIcon icon={faDoorOpen} className={Styles.guideIcon} />
+          <h5 className={Styles.guideContentTitle}>We'll Clean it.</h5>
+          <p>Our trusted & experienced maid will come to your door-step on the time for a cleaning</p>
```
Comment (zoe0119) [inline src/components/Homepage/HowItWork.jsx]: Content of guide needs to be changed.

File: src/components/Homepage/OurService.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          <FontAwesomeIcon icon={faTree} className={Styles.serviceIcon} />
+          <p className={Styles.serviceType}>Confirm Booking.</p>
+        </Col>
+        <Col xs={3} md={1}>
+          <FontAwesomeIcon icon={faPeopleCarry} className={Styles.serviceIcon} />
+          <p className={Styles.serviceType}>We'll Clean it.</p>
+        </Col>
+        <Col xs={3} md={1}>
+          <FontAwesomeIcon icon={faToilet} className={Styles.serviceIcon} />
+          <p className={Styles.serviceType}>We'll Clean it.</p>
```
Comment (zoe0119) [inline src/components/Homepage/OurService.jsx]: The kinds of services should be changed based on icon.

File: src/components/Layout/TopNav.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+const TopNav = (props) => {
+  return (
+    <div>
+      <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+        <Navbar.Brand href="#home">I<span className={Styles.ibookBrand}>book</span>ing</Navbar.Brand>
+        <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+        <Navbar.Collapse id="responsive-navbar-nav">
+          <Nav className="mr-auto">
+            <Nav.Link href="/Tasker">Tasker</Nav.Link>
```
Comment (zoe0119) [inline src/components/Layout/TopNav.jsx]: Could we change `Task` and `Tasker` here? As we have different access permission, the tasker should not access `tasker` in nav. May be changed in other content, such as service, guide.

File: src/components/Layout/TopNav.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            <Nav.Link href="/Task">Task</Nav.Link>
+          </Nav>
+          <Nav>
+            <Nav.Link href="/Login">Log in</Nav.Link>
+            <Nav.Link href="/Logout">Log out</Nav.Link>
```
Comment (zoe0119) [inline src/components/Layout/TopNav.jsx]: Is that possible that we can only see `login` when we log out, same as `log out`?

**Review by zoe0119 — CHANGES_REQUESTED** [review]

File: src/components/Homepage/HowItWork.jsx (hunk 1 above)
Comment (chrisudf) [reply src/components/Homepage/HowItWork.jsx]: done

File: src/components/Homepage/OurService.jsx (hunk 2 above)
Comment (chrisudf) [reply src/components/Homepage/OurService.jsx]: done

File: src/components/Layout/TopNav.jsx (hunk 3 above)
Comment (chrisudf) [reply src/components/Layout/TopNav.jsx]: done

File: src/components/Layout/TopNav.jsx (hunk 4 above)
Comment (chrisudf) [reply src/components/Layout/TopNav.jsx]: maybe swap log out to register rn and add islogin state latter for rendering

File: src/components/Layout/TopNav.jsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+const TopNav = (props) => {
+  return (
+    <div>
+      <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+        <Navbar.Brand href="#home">I<span className={Styles.ibookBrand}>book</span>ing</Navbar.Brand>
+        <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+        <Navbar.Collapse id="responsive-navbar-nav">
+          <Nav className="mr-auto">
+            {/* <Nav.Link href="/Post">Post</Nav.Link> */}
```
Comment (zoe0119) [inline src/components/Layout/TopNav.jsx]: Feel free to delete these comments.

File: src/components/Homepage/OurService.jsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+          <FontAwesomeIcon icon={faHouseDamage} className={Styles.serviceIcon} />
+          <p className={Styles.serviceType}>We'll Clean it.</p>
+        </Col>
+        <Col xs={3} md={1}>
+          <FontAwesomeIcon icon={faBug} className={Styles.serviceIcon} />
+          <p className={Styles.serviceType}>Deworming</p>
+        </Col>
+        <Col xs={3} md={1}>
+          <FontAwesomeIcon icon={faCar} className={Styles.serviceIcon} />
+          <p className={Styles.serviceType}>car washing</p>
```
Comment (zoe0119) [inline src/components/Homepage/OurService.jsx]: UpperCase it.

File: src/components/Homepage/OurService.jsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+          <FontAwesomeIcon icon={faPeopleCarry} className={Styles.serviceIcon} />
+          <p className={Styles.serviceType}>Moving</p>
+        </Col>
+        <Col xs={3} md={1}>
+          <FontAwesomeIcon icon={faToilet} className={Styles.serviceIcon} />
+          <p className={Styles.serviceType}>Plumber</p>
+        </Col>
+        <Col xs={3} md={1}>
+          <FontAwesomeIcon icon={faHouseDamage} className={Styles.serviceIcon} />
+          <p className={Styles.serviceType}>We'll Clean it.</p>
```
Comment (zoe0119) [inline src/components/Homepage/OurService.jsx]: 'We will clean it' seems not right?

File: src/components/Routes.jsx [hunk 8]
```diff
@@ -1,14 +1,13 @@
...
-import { BrowserRouter as Router, Route, Link, Switch } from "react-router-dom";
-import Task from './Task'
-import Tasker from './Tasker'
-import Seeker from './Seeker'
+import { BrowserRouter as  Route, Switch } from "react-router-dom";
+import Register from './Register'
+import Tasker from '../Tasker'
 import Home from './Home'
-import Routes from './Routes'
+// import Routes from './Routes'
```
Comment (zoe0119) [inline src/components/Routes.jsx]: Delete it too.

Comment (chrisudf) [reply src/components/Routes.jsx]: done

File: src/components/Homepage/OurService.jsx (hunk 7 above)
Comment (chrisudf) [reply src/components/Homepage/OurService.jsx]: edit to house repairing

File: src/components/Homepage/OurService.jsx (hunk 6 above)
Comment (chrisudf) [reply src/components/Homepage/OurService.jsx]: done

File: src/components/Layout/TopNav.jsx (hunk 5 above)
Comment (chrisudf) [reply src/components/Layout/TopNav.jsx]: done

**Review by zoe0119 — APPROVED** [review]
LGTM

File: src/components/Homepage/OurService.jsx (hunk 6 above)
Comment (zoe0119) [reply src/components/Homepage/OurService.jsx]: :)

**Comment by zoe0119** [comment]
@chrisudf The comment might be different, for example, the last one could be "uppercase service title".

**Pull request merged**